### PR TITLE
Image - make `src` to support absolute and relative

### PR DESCRIPTION
**Pull request by nitayneeman**
This PR removes `mediaItem` prop and makes `src` to support both absolute and relative sources.

### CODE REVIEW

File: src/components/Image/Image.spec.tsx [hunk 1]
```diff
@@ -10,69 +10,68 @@ import { imageDriverFactory } from './Image.driver';
...
+      const driver = createDriver(<Image src={src} />);
+
+      const mediaItemUrl = await driver.getSrc();
+
+      expect(mediaItemUrl.startsWith('http')).toBe(true);
+      expect(mediaItemUrl.endsWith(src)).toBe(true);
     });
+  });
+
+  describe.each(Object.keys(sampleSources))('%s', (sourceType) => {
```
Comment (nitayneeman) [inline src/components/Image/Image.spec.tsx:60]: I believe we'll have more tests related to both in the upcoming tasks

File: src/components/Image/Image.spec.tsx [hunk 2]
```diff
@@ -10,69 +10,68 @@ import { imageDriverFactory } from './Image.driver';
...
+  describe('relative', () => {
+    const src = sampleSources.relative;
+
+    it('should return `src` as full media item URL containing the provided URI', async () => {
+      const driver = createDriver(<Image src={src} />);
+
+      const mediaItemUrl = await driver.getSrc();
+
+      expect(mediaItemUrl.startsWith('http')).toBe(true);
+      expect(mediaItemUrl.endsWith(src)).toBe(true);
```
Comment (nitayneeman) [inline src/components/Image/Image.spec.tsx:56]: We also can install `image-client-api` and check the exact match, but I think it's good enough like that

**Comment by WixBuildServer** [comment]
### 🤖 Hi, I'm #Janet!

  A deploy preview of your PR was successfully released 🖼.

  You can find it under one of the following URLs:
  * https://bo.wix.com/pages/wix-ui-tpa.pr_551/

**Review by jonathanadler — COMMENTED** [review]
Looks great!

Just left one small comment

File: src/components/Image/Image.tsx [hunk 3]
```diff
@@ -20,42 +17,37 @@ export interface ImageProps extends TPAComponentProps {
   onLoad?: React.EventHandler<React.SyntheticEvent>;
   /** A callback to be called if error occurs while loading */
   onError?: React.EventHandler<React.SyntheticEvent>;
-  /** Exposing the MediaPlatformItem API as specified in https://wix-wix-ui-core.surge.sh/?activeTab=Usage&path=%2Fstory%2Fcomponents--media-image */
-  mediaItem?: MediaItemProps;
```
[the remark this answers is not in the document]
Comment (nitayneeman) [reply src/components/Image/Image.tsx:24]: Yeah, actually I planned to check it 👌
And a good idea, I'll move to WIP.

**Comment by nitayneeman** [comment]
@jonathanadler can we merge it?

**Review by jonathanadler — APPROVED** [review]
Yeah, sorry

**Pull request merged**